source/library/storage.js

```diff
@@ -1,82 +1,83 @@
-import { getGenericPassword, resetGenericPassword } from "react-native-keychain";
-import { Platform } from "react-native";
+// import { getGenericPassword, resetGenericPassword } from "react-native-keychain";
+// import { Platform } from "react-native";
 
-import AsyncStorageInterface from "../compat/AsyncStorageInterface";
-import SecureStorageInterface from "../compat/SecureStorageInterface";
+// import AsyncStorageInterface from "../compat/AsyncStorageInterface";
+// import SecureStorageInterface from "../compat/SecureStorageInterface";
```

Comment:
I guess this whole file could be removed?

Reply:
Good idea 👍